molo/yourwords/models.py

```diff
@@ -0,0 +1,24 @@
+from wagtail.wagtailcore.models import Page
+
+
+class competition(Page):
+    name = Page.CharField(max_length=128)
+    startDate = Page.DateTimeField(null=True, blank=True)
+    endDate = Page.DateTimeField(null=True, blank=True)
```

Comment:
You're also missing a few fields we discussed from the SoW - description and body